demos/common/python/openvino/model_zoo/model_api/adapters/utils.py

```diff
@@ -0,0 +1,71 @@
+"""
+ Copyright (C) 2022 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import re
+from typing import Optional
+from openvino.runtime import layout_helpers
+
+
+class Layout:
+    def __init__(self, layout = '') -> None:
+        self.layout = layout
+
+    @staticmethod
+    def from_shape(shape):
+        '''
+        Create Layout from given shape
+        '''
+        if len(shape) != 4:
+            raise RuntimeError('Get layout from shape method supports only 4D input shape')
+
+        layout = 'NCHW' if shape[1] in range(1, 4) else 'NHWC'
```

Comment:
Could you aling it with [cpp implementation](https://github.com/openvinotoolkit/open_model_zoo/pull/3104#discussion_r804748341), please

Reply:
Done

Reply:
You added the 2D and 3D layouts verification, however this method is called only if `len(input.shape) == 4`